llvm/tools/sycl-post-link/SpecConstants.cpp

```diff
@@ -336,21 +334,20 @@ Instruction *emitSpecConstantComposite(Type *Ty,
 Instruction *emitSpecConstantRecursiveImpl(Type *Ty, Instruction *InsertBefore,
                                            SmallVectorImpl<unsigned> &IDs,
                                            bool IsNewSpecConstant,
```

Comment:
This is no longer used - please remove

Reply:
Removed in e09b1b859bd88f8dc46b0aaad8e4bc21a7993b27